internal/database/sqlite.go

```diff
@@ -75,6 +75,10 @@ var (
 			"create_accounts",
 			`create table if not exists accounts(account_id primary key, customer_id, user_id, encrypted_account_number, hashed_account_number, masked_account_number, routing_number, status, type, created_at datetime, deleted_at datetime);`,
 		),
+		execsql(
+			"create_account_ofac_searches",
+			`create table if not exists account_ofac_searches(account_ofac_search_id varchar(40) primary key, account_id varchar(40), entity_id varchar(40), sdn_name varchar(40), sdn_type integer, percentage_match double precision (5,2), created_at datetime);`,
```

Comment:
I think this will have to be at the end of the array. If I remember right the migrations have an index and checksum that'll break if we add the migration in the middle.

Reply:
Fixed